resources/windows/wix/customactions.wxs.erb

```diff
@@ -125,11 +125,6 @@ End If
       Property="PUPPET_AGENT_ENVIRONMENT"
       Value="[CMDLINE_PUPPET_AGENT_ENVIRONMENT]"
       Execute="firstSequence" />
-    <CustomAction
-      Id="SetDefaultPuppetAgentEnvironment"
```

Comment:
Shouldn't we also delete the action from the sequence in https://github.com/puppetlabs/puppet-agent/blob/954f06b24f68ffdfa0bdb2d2b9bc73970ecb8a37/resources/windows/wix/sequences.wxs.erb#L36?

Reply:
Yip - doing that now - sorry - had submitted PR before checking build had completed.